.evergreen/.evg.yml

```diff
@@ -384,7 +384,7 @@ functions:
 # Anchors
 
 hosts: &hosts
- - rhel62-small
+ - rhel70-small
```

Comment:
Had to update to RHEL 7 to support libmongocrypt, which isn't yet built for RHEL 6